Views/home.stencil

```diff
@@ -3,27 +3,27 @@
 {% block title %}Hackers at Cambridge{% endblock %}
 
 {% block content %}
-  <header class="">
+  <header>
     <h1>Hackers at Cambridge</h1>
-    <h2 class="mono">Cambridge's student tech society</h2>
```

Comment:
We can handle this in a few ways (which are not mutually exclusive). The key thing to remember here is that `mono` does not semantically describe what the class does, it describes the style that it applies, which tightly couples the semantics to the styling. 

We're giving this text a monospaced font because of the uniform space taken up by each glyph, but to apply some kind of emphasis.

This is especially important if the person writing the markup is also not the designer (mono means something significant to designers but maybe not to others).

So the key is to make a semantic class name for this. There may be more than one of these.

For example:

- We make a general `Text--emphasis` class (this probably isn't the best name due to confusion with `<em>`, but you get the idea, that does exactly what `mono` did.
- Give this particular `h2` element a very descriptive semantic name, `.LandingIntro__tagline`, and apply the styles directly to that. We could make a mixin (maybe called `Text--emphasis`?)

And there are options in between as well.

Again, this is similar to the process you'd go with deciding to name functions/variables/classes in the rest of your code. Do we name a function based on the algorithm it uses, or what you'd actually use the function for?